.orchestra/ci/install-python-dependencies.sh

```diff
@@ -22,6 +25,10 @@ PYTHON_DEPENDENCIES=(
   "pyparsing==2.4.7"
   "pyyaml"
   "setuptools"
+  "types-backports"
+  "types-PyYAML"
+  "types-requests"
+  "types-urllib3"
```

Comment:
Are these just type hinting stuff?
How do they solve the problem of annotations and the actual package going out of sync?

Reply:
This is an unresolved problem in the python ecosystem.
The 4 packages above refer to pretty mature libraries, I don't expect API changes anytime soon